### DISCUSSION TITLE
Can Tomb1Main be played with randomizer?

### DISCUSSION THREAD

**Discussion by Menghus** [LostArtefacts/TRX#780] [Q&A] [answered]
As the title says, I wonder if Tomb1Main can be played with randomizer and if so, how to do it? Is it the same for TR2Main?

**Comment by lahm86** [comment, marked as answer]
The use of Tomb1Main in the randomizer is encouraged. Setting it up is straight-forward.

1. Create a clean installation of Tomb1Main - it's recommended to keep this separate from any normal playthrough installation that you use. Make sure the data files you use are original and not modified by any other tool.
2. Download the randomizer and point it to the data directory of the folder you created above.
3. Randomize and play.

There is a detailed guide below, which also includes a step-by-step video. If you have any issues, feel free to raise them in the [TR-Rando](https://github.com/LostArtefacts/TR-Rando) repository.

https://github.com/LostArtefacts/TR-Rando/blob/master/USING.md
